### DISCUSSION TITLE
Can't use on two windows logins?

### DISCUSSION THREAD

**Discussion by Sheldonopolis45** [aclap-dev/video-downloadhelper#928] [License & Purchase requests]
### What's your issue?

I have two logins on my Windows Laptop. I purchased the tool with my admin profile. With my second profile, it's asking me to purchase another license. Is that correct? If so I'll need a refund because I really only need it on my "non-admin" profile.

### The 10 first digits (only 10) of your license number or email address

e0201d403c

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

In Windows, each user (in the sense of how Microsoft uses the term) is treated as being on a completely different computer.  So if you want to install VDH, you need to do it on each "computer."  In other words, just open your browser when you're logged in with your administrator user & install VDH.  Then do the log in as another user trick.  Then open your browser again & install VDH again.  Register the same license number under each user.  That should set you up.  There's only 1 instance of Firefox on the computer, but there's different Firefox profiles in use with different Windows permissions depending on which Windows user you logged in as.

You don't need a refund.  You will be able to use VDH as a licensed user no matter who you log in as.  If you have more users defined, just install VDH under each user, & register the same license each time.

But beware.  There is a limit to the number of times you can install VDH with a given license before your license will become locked.  This is a simple check to prevent license sharing.  You are allowed to use the same VDH license on however many computers you own now, & on whatever computers you may own in the future.  For the rest of your life.  But you're not supposed to share your license with your friends & family.  So after a few installs, VDH locks your license.  There is a simple procedure for unlocking your license.  Click the Wiki link at the top of any page of this forum & look for the relevant discussion.

Oh crap.  It's not there.  Hey @jcfrog Jérôme, this needs to go in the Wiki.

I'm not so sure about this any more.  I just uninstalled & reinstalled my VDH about a dozen times & my license didn't lock.  I used to be able to lock my license by reinstalling a few times.  But now I can't do it.  Maybe it works differently now.  Anyway, the way it USED TO work is at a certain point, your VDH settings would say your license is locked & needs to be revalidated.  You would just click the `Recheck license` button & that would trigger an E-mail to the address you used when you purchased your license.  That E-mail would contain a link you could click & that would unlock your license.  You can see how it used to look in past releases of VDH by visiting this page:

https://groups.google.com/g/video-downloadhelper-q-and-a/c/BzPLK2YyL-s

Do a text search on that page for "locked" & you'll see screenshots of how it used to work.  I'm very surprised I couldn't get this to happen with VDH 9.0.2.12a1 beta for Firefox.  I totally have no idea what's going on.  Maybe after each uninstall of VDH, I needed to close all my browser windows.  That's too much trouble.  Anyway, if it happens to you, maybe you can collect some screenshots & provide a tutorial that can be copied into the Wiki.

**Reply by GrampaWildWilly** [reply]
I managed to get the necessary screenshots.  In connection with discussion #942, I managed to lock my license.  Perhaps all the uninstalls & reinstalls I did here had an influence.  I can't tell.  But as part of that other discussion, I entered simply x as my license key, then clicked the `Check license key` button.  That of course told me my license was not registered, which is what I was trying to do in the context of that other discussion.  However, when I pasted my license key back into the appropriate field & clicked the `Check license key` button, I got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8d7af847-c733-4bc1-82ec-9a5734366c4f)

So I followed the instruction to check my E-mail.  This is what I received:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/d441c9b5-438f-4387-9efb-39c2f956210b)

Appropriately censored, as you can see.  This is the same E-mail as ever.  When I clicked the link in the E-mail, I got sent to this web page:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/44109f4c-6281-4411-badc-77750944b575)

After that, the VDH settings already showed this:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6cc3e6b2-f08b-4edd-bb05-e2aa084bf9d2)

That's all it takes.  I didn't even have to click the `Check license key` button again.  It was already registered & working again.
